backend/de.metas.distribution.rest-api/src/main/java/de/metas/distribution/rest_api/JsonDistributionEvent.java

```diff
@@ -44,6 +44,13 @@ public Optional<Quantity> getQtyPicked(@NonNull final I_C_UOM uom)
 
 	@Nullable PickFrom pickFrom;
 
+	@Value
+	@Builder
+	@Jacksonized
+	public static class ReversePicking {}
```

Comment:
in mobile picking, we are using the "Un-pick" naming (see de.metas.picking.rest_api.json.JsonPickingStepEvent.EventType).

wondering why not keep the same terminology.